packages/@aws-cdk/aws-ecs-patterns/lib/base/queue-processing-service-base.ts

```diff
@@ -277,6 +285,7 @@ export abstract class QueueProcessingServiceBase extends CoreConstruct {
         retentionPeriod: props.retentionPeriod || Duration.days(14),
       });
       this.sqsQueue = new Queue(this, 'EcsProcessingQueue', {
+        visibilityTimeout: props.visibilityTimeout || undefined,
```

Comment:
Since the prop is optional, we don't need the `|| undefined`

```suggestion
        visibilityTimeout: props.visibilityTimeout,
```